tests/e2e/tests/05-elasti-crd-remove/00-scale-target.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: target-deployment
+  namespace: target
+spec:
+  replicas: 0
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: target-deployment
+  namespace: target
+spec:
+  replicas: 1
+---
```

Comment:
I know this is weird, but we are doing it:

- To get controller to create the private service.
- It doesn't create it unless the proxy hasn't been enabled atleast once.